docker-compose.test.yml

```diff
@@ -0,0 +1,45 @@
+---
+# Compose file for test runs
+version: "3.7"
+services:
+  db:
+    image: datarttu/sujuikodb:latest
+    build: .
+    restart: unless-stopped
+    env_file: .env_test
+    volumes:
+      - type: bind
+        source: ./db/db_test
+        target: /docker-entrypoint-initdb.d/
+        read_only: true
+      - type: tmpfs
+        target: /var/lib/postgresql/data
+    healthcheck:
+      test: ["CMD", "pg_isready", "-q", "-U", "${POSTGRES_USER}"]
+      interval: 30s
+      start_period: 5s
+      timeout: 20s
+      retries: 3
+  dataimporter:
+    image: datarttu/sujuikodb:latest
+    depends_on:
+      db:
+        condition: service_healthy
+    restart: on-failure
+    volumes:
+      - type: bind
+        source: ${IMPORT_DATA_DIR}
+        target: /data
+        read_only: true
+    env_file: .env_test
+    # TODO: See if there is a better way to prevent dataimporter from
+    #       trying imports before db is ready.
+    #       Can we utilize healthcheck of db
+    #       like docker-compose < v3 depends_on condition can do?
+    command: ["psql", "-f", "/data/import.sql"]
+    environment:
+      PGHOST: db
+      PGPORT: 5432
+      PGUSER: ${POSTGRES_USER}
+      PGDATABASE: ${POSTGRES_DB}
+      PGPASSWORD: ${POSTGRES_PASSWORD}
```

Comment:
```suggestion
```
For the life of me, I could not make the variables in `environment` use the variables from `env_file`. So `psql` would try to connect as `root`, for example. Instead, I added all of these into `.env_test` and it worked like a dream. Unfortunately that meant repeating some constants.

Reply:
👍 #41